core/arch/arm/kernel/generic_boot.c

```diff
@@ -52,6 +52,13 @@
 
 #define PADDR_INVALID		0xffffffff
 
+/*
+ * Array used when booting, to synchronize cpu.
+ * When 0, the cpu has not started.
+ * When 1, it has started
+ */
+uint32_t sem_cpu_sync[CFG_TEE_CORE_NB_CORE];
```

Comment:
Don't you need to add `__data` since this variable isn't initialized before it's used?